packages/contracts-bedrock/test/setup/ForkLive.s.sol

```diff
@@ -282,11 +287,16 @@ contract ForkLive is Deployer, StdAssertions, FeatureFlags {
         address permissionedDisputeGame = address(disputeGameFactory.gameImpls(GameTypes.PERMISSIONED_CANNON));
         artifacts.save("PermissionedDisputeGame", permissionedDisputeGame);
 
-        address permissionlessDisputeGame = address(disputeGameFactory.gameImpls(GameTypes.CANNON));
-        if (permissionlessDisputeGame != address(0)) {
-            // Both names are used in different places, so we save both.
-            artifacts.save("PermissionlessDisputeGame", address(permissionlessDisputeGame));
-            artifacts.save("FaultDisputeGame", address(permissionlessDisputeGame));
+        address cannonDisputeGame = address(disputeGameFactory.gameImpls(GameTypes.CANNON));
+        if (cannonDisputeGame != address(0)) {
+            // Store under FaultDisputeGame for backwards compatibility with superchain-registry
+            artifacts.save("FaultDisputeGame", address(cannonDisputeGame));
```

Comment:
All consumers of artifacts now use the `FaultDisputeGameCannon` key, so we can drop the alias for `FaultDisputeGame`. Neither superchain-ops nor superchain-registry rely on the deployment artifacts defined here.
While the initial forked contracts are still sourced from the registry (which uses the `FaultDisputeGame` key), the new deployment artifacts we write out don’t need to maintain that legacy schema.